src/Uno.Toolkit.UITest/Framework/ColorCodeParser.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Drawing;
+using System.Linq;
+
+namespace Uno.Toolkit.UITest.Framework
+{
+	internal static class ColorCodeParser
+	{
+		public static Color Parse(string colorCode)
+		{
+			if (colorCode == null)
+			{
+				throw new ArgumentNullException(nameof(colorCode));
+			}
+
+			if (!colorCode.StartsWith("#", StringComparison.OrdinalIgnoreCase))
+			{
+				throw new FormatException("Color code must start with #");
+			}
+
+			byte a = 0x00;
+			byte r = 0x00;
+			byte g = 0x00;
+			byte b = 0x00;
+
+			colorCode = colorCode.Substring(1);
+
+			if (colorCode.Length == 3)
```

Comment:
leave a comment on which syntax you are parsing from: rgb,argb,rrggbb,aarrggbb
mostly for 3 and 4 which arent seen as frequently

Reply:
see comment in fluentextensions.cs